lib/modules/xPostLinks.js

```diff
@@ -0,0 +1,66 @@
+addModule('xPostLinks', function(module, moduleID) {
+	module.moduleName = 'X-post Links';
+	module.category = 'Posts';
+	module.description = 'Create links to x-posted subreddits in the taglines.';
+	module.include = [
+		/^https?:\/\/([a-z]+)\.reddit\.com/i
+	];
+
+	module.go = function() {
+		if ((module.isEnabled()) && (module.isMatchURL())) {
+
+			module.createLinks();
+			window.addEventListener('neverEndingLoad', module.createLinks);
+		}
+	};
+
+	module.createLinks = function () {
```

Comment:
Very minor: `createLinks` doesn't have to be a module property, it can just be `function createLinks() {...}`